### DISCUSSION TITLE
probably caused by bad db migrations. I use op. can't start backend

### DISCUSSION THREAD

**Discussion by qianyiwangmeng** [blockscout/blockscout#9988] [Q&A]
![image](https://github.com/blockscout/blockscout/assets/94897566/23459b82-1da7-4ae8-ba69-237dfa087199)
![image](https://github.com/blockscout/blockscout/assets/94897566/794d4cb9-9e29-4a9c-8348-2d9b7c46a44a)

**Comment by sl1depengwyn** [comment]
Please, provide some more details on your setup, is it docker or manual build? It seems that you haven't set `CHAIN_TYPE` env var
